unit_test/tstGridOperator2d.hpp

```diff
@@ -0,0 +1,196 @@
+/****************************************************************************
+ * Copyright (c) 2021 by the Picasso authors                                *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Picasso library. Picasso is distributed under a *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Picasso_FieldManager.hpp>
+#include <Picasso_FieldTypes.hpp>
+#include <Picasso_GridOperator.hpp>
+#include <Picasso_InputParser.hpp>
+#include <Picasso_ParticleInit.hpp>
+#include <Picasso_ParticleInterpolation.hpp>
+#include <Picasso_ParticleList.hpp>
+#include <Picasso_Types.hpp>
+#include <Picasso_UniformCartesianMeshMapping.hpp>
```

Comment:
Plan is to wait to transition the 3D test until the old mesh is deprecated?

Reply:
Right. The 2d test is also lacking the particle component which will need to be updated for 2d